cardano-node/ChangeLog.md

```diff
@@ -1,5 +1,25 @@
 # Changelog for cardano-node
 
+## 1.36.0 -- November 2022
+
+### node changes
+
+ - Bumped cardano-ledger to 0.1
+ - Bumped plutus-core to 1.0
+ - Bumped ouroboros-network to commit 72863b0
+
+### consensus changes
+
+TODO
```

Comment:
?

Reply:
There's a [comment](https://github.com/input-output-hk/cardano-node/pull/4608#discussion_r1037038649) on that already. We're not making a release from this PR directly, so we'll fill out the changelog once everything's in place.